src/schema/validators/common.ts

```diff
@@ -321,10 +321,7 @@ export const authorItem = {
             type: 'string',
             contentType: 'text',
         },
-        avatar: {
-            type: 'string',
-            pattern: imageUrlPattern,
-        },
+        avatar: ImageProps,
```

Comment:
Could it be a breaking change?

Reply:
no, it actually expands the use cases as ImageProps contains previous schema:
<img width="394" alt="Screenshot 2023-10-23 at 12 18 25" src="https://github.com/gravity-ui/page-constructor/assets/30153778/300d00ed-adf5-49ba-9c42-39da835de333">